web/packages/extension/tools/sign_xpi.js

```diff
@@ -54,4 +57,8 @@ async function sign(
             "Skipping signing of Firefox extension. To enable this, please provide MOZILLA_API_KEY, MOZILLA_API_SECRET and FIREFOX_EXTENSION_ID environment variables"
```

Comment:
I wanted to comment few lines above but GitHub doesn't allow.
```js
const { version } = require("../assets/manifest.json");
```
Can be rewritten to:
```js
const version = process.env.npm_package_version;
```

Reply:
Actually we need the version from the manifest, because it has the build version appended (`0.1.0.20120415`), and this is necessary for the Mozilla add-on service to accept it (version # must be unique). There may be a better way to pass this into the signing code, though.